datastore/migrations/core/migrater.py

```diff
@@ -38,7 +38,10 @@ def to_position_data(self) -> PositionData:
 @service_interface
 class Migrater(Protocol):
     def migrate(
-        self, target_migration_index: int, migrations: Dict[int, BaseMigration]
+        self,
+        target_migration_index: int,
+        migrations: Dict[int, BaseMigration],
+        start_migration_index: int = 0,
```

Comment:
I would prefer to leave the interface as-is and instead set the start migration index in the `set_additional_data` method. I think this is more fitting

Reply:
Done